### PR TITLE
Move bower package management.  Upgrade to latest version of Backbone.js

### PR DESCRIPTION
**Pull request by jbarnicle**
Requires npm install.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -2,8 +2,5 @@
   "name": "regulations-site",
```
Comment (theresaanna) [inline bower.json]: Do we need to keep the `bower.json` at all?

Comment (jbarnicle) [inline bower.json]: Nope.  I'll remove it.

**Comment by cmc333333** [comment]
This is stalled while respond.js gets re-added

**Comment by cmc333333** [comment]
@jbarnicle does html5shiv need to be re-added, too?

**Comment by jbarnicle** [comment]
@cm - html5shiv is not referenced anywhere - altho it looks like a version might be inlined along with modernizr in base.html.  Also it turns out that respond.js is already present in the -site git repository (static/regulations/js/built_ and doesn't need to be retrieved via any package manager.   I am doing some double checking but this PR might be good to go as is.

**Comment by cmc333333** [comment]
All ayes say aye

**Pull request merged**